ert/ensemble_evaluator/builder/_ensemble.py

```diff
@@ -71,10 +71,8 @@ async def send_cloudevent(  # pylint: disable=too-many-arguments
         cert: Optional[Union[str, bytes]] = None,
         retries: int = 1,
     ) -> None:
-        client = Client(url, token, cert)
-        await client._send(to_json(event, data_marshaller=evaluator_marshaller))
-        assert client.websocket  # mypy
-        await client.websocket.close()
+        async with Client(url, token, cert, max_retries=retries) as client:
```

Comment:
Based on oral discussion: `max_retries` is set to 10 in Client init, therefore if we want to keep the current behaviour `retries` should be set to 10

Reply:
fixed!